app/components/GettingStartedModal/localStorage.ts

```diff
@@ -1,8 +1,6 @@
-export const isGettingStartedModalDisabledInLocalStorage = (): boolean => {
-    const modalDisabled = localStorage.getItem('getting-started-modal-disabled');
-    return modalDisabled === undefined || modalDisabled !== 'true';
-};
+export const isGettingStartedModalDisabled = (): boolean =>
+    localStorage.getItem('getting-started-modal-disabled') === 'true';
```

Comment:
Optional:

Extracting the key and reusing it in both function would ensure there are no typos :slightly_smiling_face:

Reply:
Actually, since the value is also a string (`'true'`) it could be extracted and reused as well, as it is prone to the same 'typo issue'.

Reply:
Yeah :smile: We could alternatively use a different value, not just `true`, as seeing a boolean used as a string could be misleading :slightly_smiling_face: To me, any truthy value would work, so we could do

```suggestion
    !!localStorage.getItem('getting-started-modal-disabled');
```

Reply:
It is good, as 'yes' or '1' would work as well, for example if the value was set manually or in a different place, like a test.
It is bad, as 'no', 'false' or '0' would be considered a truthy value.
I think this is actually a common issue wherever string must be used to describe boolean value. Maybe there is something in npm that could solve it by recognizing common string values that can be used to express true or false?

Reply:
String(true) could be some solution too

Reply:
Yeah :thinking: or using `JSON.stringify` on the value. Or, we could avoid a boolean value and use an enum https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/#3-if-it-makes-sense-use-an-enum

```ts
enum GettingStartedModalState {
  Disabled = 'disabled',
  Enabled = 'enabled',
}
```

(we could use `dismissed` instead of `disabled`), and use other key names, e.g. `getting-started-modal-dismissed-state`.

That's actually one of the ways to avoid booleans that is slightly more descriptive. 

I don't have a preference here, just putting it out there :slightly_smiling_face:

Reply:
@gregcfy So, what's your take on this? You re-requested the review but this thread is still open. The comments here are optional, but it would be nice to know if you are planning any actions or not 🙂.

Reply:
I have decided to use JSON.stringify(true)